app/setup.cfg

```diff
@@ -11,4 +11,4 @@ skip_glob = */migrations/*.py
 DJANGO_SETTINGS_MODULE = peopledepot.settings
 python_files = tests.py test_*.py *_tests.py
 # addopts = -vv -x --last-failed --cov --cov-report html
```

Comment:
Oh, that's commented out because we don't use it. I included the line as some options that might be useful in some cases but not used. It also shows the syntax to use for adding options.